core/src/avm2/object/class_object.rs

```diff
@@ -87,10 +87,6 @@ pub struct ClassObjectData<'gc> {
 }
 
 impl<'gc> ClassObject<'gc> {
-    pub fn name(self) -> QName<'gc> {
-        self.inner_class_definition().read().name()
-    }
-
```

Comment:
Was this unused?

Reply:
Yeah. I added it while print-debugging the vtable PR and forgot to remove it; not saying it's not useful, but let's add it when it actually becomes useful :)